go-dlv.el

```diff
@@ -107,7 +107,7 @@
 
     output))
 
-(defcustom go-dlv-command-name "dlv"
+(defcustom gud-dlv-command-name "dlv"
```

Comment:
this would break every user's setup that customized this variable, so best to avoid.

Reply:
Thanks for your quick response. `gud-query-cmdline` uses `(gud-val 'command-name minor-mode)` to lookup for the command which: This function in turn resolves the name using the `gud-dlv` prefix:


> this would break every user's setup that customized this variable, so best to avoid.
Yes I also thought about this. Maybe a alias would be more more appropriate.

Reply:
Cool, thx. Can you please add a comment like this above this line, or in the commit message? Then I'll merge.

Reply:
Of course. I amended the commit message and also added the alias.

Reply:
Excellent